cmd/cli/user_test.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/moira-alert/moira/database/redis"
+	"github.com/moira-alert/moira/logging/go-logging"
+
+	"github.com/moira-alert/moira"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestUpdateUsers(t *testing.T) {
+	conf := getDefault()
```

Comment:
As I can see this test is an integration test and requires connection to Redis. I propose to skip tests like this in short mode.

```go
    if testing.Short() {
        t.Skip("skipping test in short mode.")
    }
```

Reply:
These are not integration tests.

Reply:
Still this tests are not unit tests and require additional dependencies and should be skipped if I run tests in short mode.

Reply:
added